application/src/main/java/org/togetherjava/tjbot/commands/moderation/attachment/AttachmentListener.java

```diff
@@ -0,0 +1,118 @@
+package org.togetherjava.tjbot.commands.moderation.attachment;
+
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.MessageBuilder;
+import net.dv8tion.jda.api.entities.Message;
+import net.dv8tion.jda.api.entities.MessageEmbed;
+import net.dv8tion.jda.api.events.message.MessageReceivedEvent;
+import net.dv8tion.jda.api.requests.RestAction;
+import org.jetbrains.annotations.NotNull;
+import org.jooq.tools.StringUtils;
+import org.togetherjava.tjbot.commands.MessageReceiverAdapter;
+import org.togetherjava.tjbot.config.Config;
+import org.togetherjava.tjbot.moderation.ModAuditLogWriter;
+
+import java.awt.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Pattern;
+
+/**
+ * Listener that receives all sent messages in every channel and checks if the message has any
+ * blacklisted file attached.
+ * <p>
+ * If there was a blacklisted file attached, delete the message and send the user a dm, telling what
+ * they did wrong and also inform the mods.
```

Comment:
Imo this description is too detailed/implementation specific.

It might be enough to write sth along the lines of:

> Reacts to blacklisted attachmenents being posted, upon which they are deleted.